moveit_ros/moveit_servo/config/servo_parameters.yaml

```diff
@@ -22,58 +45,32 @@ servo:
                   must be passed to the node during launch time."
   }
 
-  monitored_planning_scene_topic: {
-    type: string,
-    default_value: "/planning_scene",
-    description: "The name of the planning scene topic. \
-                  planning_scene_monitor::PlanningSceneMonitor::DEFAULT_PLANNING_SCENE_TOPIC"
-  }
-
-################################ OUTPUTS #########################################
-  status_topic: {
-    type: string,
-    default_value: "~/status",
-    description: "The topic to which the status will be published"
-  }
-
-  command_out_topic: {
-    type: string,
-    default_value: "/panda_arm_controller/joint_trajectory",
-    description: "The topic on which servo will publish the joint trajectory \
-                  Change this to the topic your controller requires."
-  }
-
-  command_out_type: {
-    type: string,
-    default_value: "trajectory_msgs/JointTrajectory",
-    description: "The type of command that servo will publish",
-    validation: {
-      one_of<>: [["trajectory_msgs/JointTrajectory", "std_msgs/Float64MultiArray"]]
-    }
-  }
-
-################################ INPUTS  #############################
+############################# INCOMING COMMAND SETTINGS ########################
 
   pose_command_in_topic: {
     type: string,
+    read_only: true,
     default_value: "~/pose_target_cmds",
     description: "The topic on which servo will receive the pose commands"
   }
 
   cartesian_command_in_topic: {
     type: string,
+    read_only: true,
     default_value: "~/delta_twist_cmds",
     description: "The topic on which servo will receive the twist commands"
   }
 
   joint_command_in_topic: {
     type: string,
+    read_only: true,
     default_value: "~/delta_joint_cmds",
     description: "The topic on which servo will receive the joint jog commands"
   }
 
   command_in_type: {
     type: string,
+    read_only: true,
```

Comment:
Why should this be read only? Far as I know, this is just an if statement in the code so pretty easy to tune at runtime

Reply:
Ah yes, this shouldn't be read-only,